MidiBtlePairing/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 36
 
     defaultConfig {
         applicationId "com.mobileer.example.midibtlepairing"
         minSdkVersion 23
-        targetSdkVersion 33
+        targetSdkVersion 36
         versionCode 5
```

Comment:
I think the versionCode and versionName should be bumped even if the changes are minimal. Otherwise the app updating process may not work properly.